se/most_common_ai_questions.txt

```diff
@@ -0,0 +1,10 @@
+What is $D_i$ in "Common Principal Components Analysis"? (Occurrences: 2)
```

Comment:
This is not the code, but the output file. Please add to .gitignore. 
I would suggest to create a separate folder output/. Add it to .gitignore and then save there as many output files as we need without worrying about the repository